lib/Doctrine/ORM/EntityManager.php

```diff
@@ -237,7 +237,9 @@ public function transactional($func)
             $this->flush();
             $this->conn->commit();
 
-            return $return ?: true;
+            // Can't use ?? because PHP 7.1 is supported
+            //phpcs:ignore SlevomatCodingStandard.ControlStructures.RequireNullCoalesceOperator
+            return $return === null ? true : $return;
```

Comment:
This description looks wrong now:

https://github.com/doctrine/orm/blob/551018136bc0c4f53f047a33f7fdac128589c6b7/lib/Doctrine/ORM/EntityManagerInterface.php#L83

Also, BC-breaks should be documented in UPGRADE.md

Reply:
done